protofuzz/pbimport.py

```diff
@@ -92,11 +92,16 @@ def _compile_proto(full_path, dest):
 
 
 def from_file(proto_file):
-    """Take a filename |protoc_file|, compile it via the Protobuf compiler, and import the module.
+    """Takes either a |protoc_file| or a generated |module_file|
+    If given a `_pb2.py` file, this will try to just import the module. This should be the output of the Protobuf compiler
```

Comment:
```suggestion
    If given a `_pb2.py` file, this will try to just import the module. This should be the output of the Protobuf compiler; users should not attempt to import arbitrary Python files.
```